src/test/java/co/rsk/bitcoinj/script/StandardRedeemScriptParserTest.java

```diff
@@ -30,30 +30,30 @@ public void setUp() {
     }
 
     @Test
-    public void findKeyInRedeem_fast_bridge_redeem_script() {
-        byte[] data = Sha256Hash.of(new byte[]{1}).getBytes();
-        Script fastBridgeRedeemScript = RedeemScriptUtils.createFastBridgeRedeemScript(
-            data,
-            btcECKeyList
+    public void findKeyInRedeem_whenFlyoverRedeemScript_ok() {
```

Comment:
Should this be part of this class? Seems flyover related not standard redeem script related

Reply:
You right. I'll move them

Reply:
Done